SECURITY.md

```diff
@@ -6,55 +6,64 @@ If you believe you have found a security vulnerability in any Magma Core Foundat
 
-3. Create a new [security advisory](https://github.com/magma/magma/security/advisories/new).
-   One must be one of the repo admins to do this. Vulnerability management team members who are not
-   also a repo admin will reach out to the repo admins until they find one who can create the advisory.
-   The repo admins who are also vulnerability management team members are @tbd.
-4. [Add the reporter](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/adding-a-collaborator-to-a-security-advisory)
-   to the security advisory so that they can get updates.
-5. Inform the relevant Magma Codeowners, adding them to the security advisory.
+2. Use GitHub to create a [new security advisory](https://github.com/magma/magma/security/advisories/new). For more information on security advisories, see [the GitHub documentation](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/managing-security-vulnerabilities-in-your-project).
+ 
+    - One must be one of the repo admins to do this. Vulnerability management team members who are not also a repo admin will reach out to the repo admins until they find one who can create the advisory. 
 
-As the fix is being developed, they will then reach out to the reporter to ask them if they would like to be involved and whether they would like to be credited. For credit, the GitHub security advisory UI has a field that allows contributors to be credited.
+3. [Add the reporter](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/adding-a-collaborator-to-a-security-advisory) to the security advisory so that they can get updates.
 
-When the issue is resolved, they will contact the Magma release team and Magma Core Foundation's Outreach Committee to coordinate the publication of the security advisory.
+4. Assign an engineering resource to the ticket to implement the fix. Identify and inform the relevant Magma Codeowners, adding them to the security advisory. 
 
+5. As the fix is being developed, reach out to the reporter to ask them if they would like to be involved and whether they would like to be credited. For credit, the GitHub security advisory UI has a field that allows contributors to be credited.
 
-For more information on security advisories, see [the GitHub documentation](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/managing-security-vulnerabilities-in-your-project).
+6. When the issue is resolved, contact the Magma release team and Magma Core Foundation's Outreach Committee to coordinate the publication of the security advisory.
 
 ## Security Vulnerability Team Members
 
-The Security Vulneratily team members shall include the members of the TSC and other current Codeowners of the Magma Project as approved by the TSC.
-
-## Preferred Languages
-
-We prefer all communications to be in English.
```

Comment:
What is the reasoning behind removing the "Preferred Language" section?

Reply:
The document is now organized into a section for submitters and a section for responders. The preferred language has been pulled up into the section for submitters.